src/erlcloud_ec2.erl

```diff
@@ -32,6 +32,10 @@
          describe_snapshot_attribute/2, describe_snapshot_attribute/3,
          describe_snapshots/0, describe_snapshots/1, describe_snapshots/2,
          describe_snapshots/3, describe_snapshots/4,
+         describe_snapshots_paged/0, describe_snapshots_paged/1,
+         describe_snapshots_paged/2, describe_snapshots_paged/3,
+         describe_snapshots_paged/4,
+         next_snapshots_page/0,
```

Comment:
Hi Sergey,

I'd suggest to follow the common pattern  and name this function like `describe_snapshots_next_page()`. Same for other functions below.